dotcom-rendering/src/types/editionsCrossword.ts

```diff
@@ -0,0 +1,43 @@
+export type FEEditionsCrosswords = {
+	quick: FEEditionsCrossword;
+	cryptic: FEEditionsCrossword;
+};
+
+type FECrosswordEntry = {
+	id: string;
+	number: number;
+	humanNumber: string;
+	direction: 'across' | 'down';
+	position: { x: number; y: number };
+	separatorLocations: {
+		','?: number[];
+		'-'?: number[];
+	};
+	length: number;
+	clue: string;
+	group: string[];
+	solution: string;
+	format: string;
+};
+
+type FECrosswordDimensions = {
+	cols: number;
+	rows: number;
+};
+
+export type FEEditionsCrossword = {
+	name: string;
+	type: string;
```

Comment:
Is this value not `crosswordType: string`. We have also created this type in the [@guardian/react-crossword](https://github.com/guardian/csnx/blob/main/libs/%40guardian/react-crossword/src/%40types/CAPI.ts). It might be a good idea to export that and use it here. That way we only have the type declared in one place and it the crosswords can be passed straight into the player. Not helpful that we are currently not publishing it. I will see if we can get it published ASAP.

Reply:
See the comment below.